packages/roosterjs-content-model-dom/lib/domToModel/processors/tableProcessor.ts

```diff
@@ -200,8 +200,14 @@ export const tableProcessor: ElementProcessor<HTMLTableElement> = (
                                     colSpan <= td.colSpan;
                                     colSpan++, targetCol++
                                 ) {
-                                    for (let rowSpan = 1; rowSpan <= td.rowSpan; rowSpan++) {
-                                        const hasTd = colSpan == 1 && rowSpan == 1;
+                                    for (
+                                        let rowSpan = 1;
+                                        // RowSpan of 0 means it should span to the end of the table
+                                        // https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/td#rowspan
+                                        rowSpan <= (td.rowSpan == 0 ? 1 : td.rowSpan);
```

Comment:
If we treat it as 1, how can it extends until the end of the section?

And do we need to fix colSpan?

Any unit test here?

Reply:
The write back result will keep the rowSpan as 0, so HTML will handle that. This check here is just to make sure we process the cell and it is added to the model.

For colspan 0 is invalid, so td.colSpan returns 1 when the attribute is set as 0 for Edge, but I think we can still add a check in case other browsers handle the attribute differently.

Adding unit tests.